docs/src/PolyhedralGeometry/Polyhedra/auxiliary.md

```diff
@@ -41,6 +41,12 @@ g_vector(P::Polyhedron)
 h_vector(P::Polyhedron)
 ```
 
+## Symmetries
+```@docs
+linear_symmetries(P::Polyhedron)
+combinatorial_symmetries(P::Polyhedron)
```

Comment:
Is this intentionally in here? (I don't mind either way, just wondering)

Reply:
For now. The functions existed and I added docstrings, then I was told that everything should be done differently. I will work on this in another PR.